pkg/hintrunner/zero/zerohint_usort.go

```diff
@@ -1,14 +1,21 @@
 package zero
 
 import (
+	"fmt"
+
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
 func newUsortEnterScopeHint() hinter.Hinter {
 	return &GenericZeroHinter{
 		Name: "UsortEnterScope",
 		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> last_pos = 0
+			//> positions = positions_dict[ids.value][::-1]
+
```

Comment:
This is not the correct code for this hint. The one corresponding to this hint should be this [one](https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/usort.cairo#L15)

Reply:
I just noticed you didn't add the code in the constructor of the `UsortVerify` hint, so it was probably that you wrote it in this method by mistake. Could you please add the corresponding codes in both functions?